src/main/java/com/sequenceiq/cloudbreak/service/cluster/AmbariClusterService.java

```diff
@@ -181,11 +181,10 @@ public ClusterStatusUpdateRequest updateStatus(Long stackId, StatusRequest statu
                 cluster.setStatus(Status.START_REQUESTED);
                 clusterRepository.save(cluster);
             } else {
-                if (!Status.STOPPED.equals(clusterStatus)) {
+                if (clusterNotReadyForStart(clusterStatus)) {
                     throw new BadRequestException(
                             String.format("Cannot update the status of cluster '%s' to STARTED, because it isn't in STOPPED state.", clusterId));
-                }
-                if (!Status.AVAILABLE.equals(stackStatus)) {
+                } else if (!Status.AVAILABLE.equals(stackStatus)) {
```

Comment:
If we use 'else if' statement the cluster start could be triggered when the stack does not have 'AVAILABLE' status.